src/main.rs

```diff
@@ -19,13 +21,16 @@ use std::boxed::Box;
 use std::thread;
 use rustc_serialize::json;
 use fanout::{Fanout, Message, OpCode};
+use cron::CronSchedule;
+use chrono::*;
 
 #[derive(RustcDecodable, Eq, PartialEq, Clone, Debug)]
 struct Config { // TODO: Rename fields
     teamcity: teamcity::TeamcityCredentials,
     bitbucket: bitbucket::BitbucketCredentials,
     telegram: Option<telegram::TelegramCredentials>,
     run_interval: u64,
+    cron_interval: String,
```

Comment:
It can be done but as of this moment, I'm not sure how I should handle prioritising between `run_interval` and `cron_interval`. Just testing the feasibility of the `cron_interval` code first before I try to utilize both.